### PR TITLE
The search function should also be able to retreive data from multiple calls

### PR DESCRIPTION
**Pull request by SanyM**
Hello, 
I added the getAll function to the search function in order to retreive all the results and not only the 10 first.

### CODE REVIEW

File: lib/bot.js [hunk 1]
```diff
@@ -406,17 +406,17 @@ module.exports = (function() {
...
-			});
+        search: function(keyword, callback) {
+			this.getAll(
+				{
+					action: 'query',
+                    list: 'search',
+                    srsearch: keyword,
+                    srprop: 'timestamp'
+				},
+				'search',
```
Comment (macbre) [inline lib/bot.js]: There's something wrong with an indentation here. Please use tabs.

**Comment by macbre** [comment]
@SanyM, thanks for the change. One comment left.

**Comment by SanyM** [comment]
Hello, 
I corrected the indentation, but don't see any left comment !!

**Comment by macbre** [comment]
Thanks!

**Pull request merged**